samples/bluetooth/radio_coex_3wire/src/main.c

```diff
@@ -0,0 +1,286 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stddef.h>
+#include <sys/printk.h>
+#include <console/console.h>
+#include <sys/__assert.h>
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/uuid.h>
+
+#include <hal/nrf_radio.h>
+#include <helpers/nrfx_gppi.h>
+#include <nrfx_timer.h>
+#include <nrfx_gpiote.h>
+#include <hal/nrf_egu.h>
+#include <nrfx_ppi.h>
+
+#define gppi_channel_t        nrf_ppi_channel_t
+#define gppi_channel_alloc    nrfx_ppi_channel_alloc
+#define gppi_channel_enable   nrfx_ppi_channel_enable
+#define gppi_channel_disable  nrfx_ppi_channel_disable
+#define APP_REQ_GPIO_PIN      28 // input to app
+#define APP_GRANT_GPIO_PIN    26 // output from app
+#define EGU0_IRQn             SWI0_EGU0_IRQn
+
+#define APP_GRANT_TIMER       NRF_TIMER2
+#define APP_GRANT_TIMER_ID    2
+#define APP_GRANT_TIMER_CC0   0
+
+#define EGU_INT_PRIO          4
+
+#define COEX_NODE DT_COMPAT_GET_ANY_STATUS_OKAY(sdc_radio_coex_three_wire)
```

Comment:
I'm still seeing DT_COMPAT_GET_ANY_STATUS_OKAY here and elsewhere instead of dereferencing the phandle.

Reply:
git force push kerkuffle, it should be resolved now

Reply:
Gotcha. If you haven't heard of it already, `git push --force-with-lease` can help avoid stepping on other people's toes and I generally like it a bit better than a straight `git push --force`.